### PR TITLE
Made performKeyToDegree much closer to inverse of performDegreeToKey 

### PR DESCRIPTION
**Pull request by triss**
See issue #1163

### CODE REVIEW

**Comment by telephon** [comment]
any news on this? Do you have a simple test file?

**Comment by crucialfelix** [comment]
It sounds like this changes behavior, and this is a very musical method that is often used. We can't change it without unit tests that confirm that we won't have herds of angry musicians telling us that we embarrassed themselves in public (because they updated the night before a gig)

**Comment by jamshark70** [comment]
If this changes behavior, I'd share Felix's concern. I'd suggest taking the time to draw up a reasonably complete feature spec before merging anything. "Make it behave more like an inverse operation" is awfully vague and invites dispute... but if we have a spec, then there's something written down to explain to current and future users why it's this way and not another way.

SC development almost never begins with gathering requirements and speccing features... fine until something breaks. 

It would be nice to do without the temporary array. This is basically a math op, which should avoid unnecessary load on the garbage collector.

File: SCClassLibrary/Common/Collections/SequenceableCollection.sc [hunk 1]
```diff
@@ -594,7 +594,7 @@ SequenceableCollection : Collection {
 		^if(accidental == 0) { baseKey } { baseKey + (accidental * (stepsPerOctave / 12.0)) }
 	}
 
-	performKeyToDegree { | degree, stepsPerOctave = 12 |
+	performKeyToDegree { |key stepsPerOctave = 12|
```
Comment (telephon) [inline SCClassLibrary/Common/Collections/SequenceableCollection.sc]: comma?

**Pull request closed without merging**